src/systemcmds/mtd/mtd.c

```diff
@@ -303,6 +307,17 @@ mtd_start(char *partition_names[], unsigned n_partitions)
 
 		/* Create the partition */
 
+#ifdef CONFIG_ARCH_BOARD_AEROCORE2
```

Comment:
This should be rebased on this PR and the partition table feature being used there:
https://github.com/PX4/Firmware/pull/6400